mobile/src/events/Schedule.tsx

```diff
@@ -130,15 +186,27 @@ const Schedule: React.FC<{}> = ({ navigation }) => {
           </View>
         ) : (isToggledToUser ? events : events).length > 0 ? (
           <View>
-            {(isToggledToUser ? events : events).map((event: Event) => (
-              <EventInSchedule
-                event={event}
-                showName={!isToggledToUser}
-                navigation={navigation}
-                key={event.id}
-                avatar={isToggledToUser ? user.avatar : map[event.ownerId]}
-              />
-            ))}
+            {processedEvents && processedEvents.length != 0 &&
+
```

Comment:
remove empty line. there's some formatting issues in this PR that could prob be fixed with yarn run formatAll :)